src/pages/admin/data.hbs

```diff
@@ -1,10 +1,13 @@
 <h3>
   Download bookmarks
 </h3>
-<p>
-  You can download your bookmarks sqlite db
-  <a href="/admin/bookmarks.db">here</a>.
+<p style='margin: 0'>
```

Comment:
I added a margin change here just because I thought there was a lot of space :shrug: . Can definitely remove if desired.

Reply:
no problem with this, the only comment I have is that to be consistent with the rest of the HTML code, switch over to `"` instead of `'` (and throw a `;` on the end of the style value)

`<p style="margin: 0;">`

I think overall the web UI piece needs a lot more work, but this is fine from my POV.

Reply:
yeah, the web UI is definitely still rough. I have a couple talented friends who have offered to help out with this if they can find the time, but obviously we're all volunteering what we have to give here (and I'm still honored by how much time y'all have spent on this already). I'm fine with little stopgaps here and there with the understanding that we're probably going to blow up these templates and rebuild them.